apps/gitness/src/pages/pull-request/pull-request-changes-page.tsx

```diff
@@ -200,7 +204,7 @@ export default function PullRequestChangesPage() {
         active={''}
         loading={loadingReviewers}
         currentUser={currentUser ?? {}}
-        pullRequestMetadata={pullReqMetadata}
+        pullRequestMetadata={pullReqMetadata ? pullReqMetadata : undefined}
```

Comment:
What falsey thing can `pullReqMetaData` be that's not `undefined`?

Reply:
so this has to do with this issue on the swagger
https://harness.atlassian.net/browse/PIPE-23359 as mentioned above, this was just a workaround for the moment